v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -104,5 +104,7 @@ def create_table(self) -> hl.MatrixTable:
                 self.sample_type,
             )
         return mt.annotate_globals(
+            callset_path=self.callset_path,
```

Comment:
Makes it more clear what the source of the callset was.

Reply:
this will be the path passed in the DAG, yeah ?

Reply:
yep